src/mammos_entity/io.py

```diff
@@ -488,17 +535,26 @@ def _entities_from_csv(filename: str | Path) -> EntityCollection:
         version = re.search(r"v\d+", file_version_information)
         if not version:
             raise RuntimeError("File does not have version information in line 1.")
-        if version.group() not in ["v1", "v2"]:
+        if version.group() not in ["v1", "v2", "v3"]:
```

Comment:
For our tests we should collect files (we could write them on the fly) for all different file format versions and test that we can read all of these.